### PR TITLE
Text alignment atomics

### PR DESCRIPTION
**Pull request by ollips**
Task: task-697019

Link: preview-456

[Typography page](https://design.docs.microsoft.com/pulls/456/atomics/typography.html)

Adding text alignment atomics.

## Testing

1. Check out the code
2. Review the typography page

### CODE REVIEW

**Review by homing1 — APPROVED** [review]

File: css/src/atomics/typography.scss [hunk 1]
```diff
@@ -161,3 +161,17 @@
 .text-transform-uppercase {
 	text-transform: uppercase !important;
 }
+
+// Alignment
+
+.text-align-left {
```
Comment (wibjorn) [inline css/src/atomics/typography.scss:167]: Default, needed?

Comment (ollips) [reply css/src/atomics/typography.scss:167]: There are references in docs that need this class.

**Review by homing1 — APPROVED** [review]

File: css/src/atomics/typography.scss (hunk 1 above)
Comment (wibjorn) [reply css/src/atomics/typography.scss:167]: Mind explaining the use case?

Comment (ollips) [reply css/src/atomics/typography.scss]: Discussed offline, removed left alignment class, including on tablet.

**Review by BenDMyers — APPROVED** [review]

**Review by homing1 — APPROVED** [review]

**Review by wibjorn — APPROVED** [review]

**Pull request merged**